gnmlibrarytool/plugin.py

```diff
@@ -122,6 +121,11 @@ def __init__(self):
         self.plugin_guid = "2b849f0f-0509-4edb-98a0-78da05e60682"
 
     def return_string(self,tagname,*args):
-        return {'guid': self.plugin_guid, 'template': 'gnmlibrarytool/mediaview_storagerules.html'}
+        from models import LibraryStorageRule
+        rules = LibraryStorageRule.objects.order_by("storagerule_name")
+        context = args[1]
+        item = context['item']
+        iid = item.getId()
+        return {'guid': self.plugin_guid, 'template': 'gnmlibrarytool/mediaview_storagerules.html', 'context': {'rules': rules, 'item_id': iid}}
```

Comment:
documentation in the media.html template says

> There is a lot of context on this page, please use the debug tag if you wish to see it all.

not very useful.

But there are examples in there that it looks like you can do {{ item.getId }} directly in the template, and can also do item.getTitle.  Also some interesting stuff around ACLs; around line 18 or so.